websites/V/vscode.dev/presence.ts

```diff
@@ -0,0 +1,626 @@
+		".ins": { image: "tex" },
+		".lbx": { image: "tex" },
+		".mkiv": { image: "tex" },
+		".mkvi": { image: "tex" },
+		".mkii": { image: "tex" },
+		".texi": { image: "tex" },
+		"/^hyphen(ex)?\\.(cs|den|det|fr|sv|us)$/": { image: "tex" },
+		"/\\.te?xt$/i": { image: "text" },
+		".rtf": { image: "text" },
+		"/\\.i?nfo$/i": { image: "text" },
+		".msg": { image: "text" },
+		"/\\.(utxt|utf8)$/i": { image: "text" },
+		".toml": { image: "toml" },
+		".travis.yml": { image: "travis" },
+		".ts.map": { image: "tsmap" },
+		"/.*\\.d\\.ts/i": { image: "typescript-def" },
+		".ts": { image: "ts" },
+		".tsx": { image: "tsx" },
+		".twig": { image: "twig" },
+		".v": { image: "v" },
+		".vh": { image: "v" },
+		".vala": { image: "vala" },
+		".vapi": { image: "vala" },
+		".vb": { image: "vb" },
+		".vbs": { image: "vb" },
+		".vbhtml": { image: "vb" },
+		".vbproj": { image: "vb" },
+		".vba": { image: "vba" },
+		".vcxproj": { image: "vcxproj" },
+		".vscodeignore": { image: "vscodeignore" },
+		".vue": { image: "vue" },
+		".wat": { image: "wasm" },
+		".wast": { image: "wasm" },
+		".wasm": { image: "wasm" },
+		".xml": { image: "xml" },
+		"/\\.ya?ml$/i": { image: "yaml" },
+		"/^yarn(\\.lock)?$/i": { image: "yarn" },
+		".yarnrc": { image: "yarn" },
+		".zig": { image: "zig" },
+		"/\\.(tfvars|tf)$/i": { image: "terraform" },
+	};
+
+presence.on("UpdateData", async () => {
+	const [
+			detailIdling,
+			customDetail,
+			customState,
+			customSmallText,
+			customEmpty,
+			timestamps,
+			buttons,
+		] = await Promise.all([
+			presence.getSetting<string>("customIdling"),
+			presence.getSetting<string>("customDetail"),
+			presence.getSetting<string>("customState"),
+			presence.getSetting<string>("customImageText"),
+			presence.getSetting<string>("customEmpty"),
+			presence.getSetting<boolean>("timeStamps"),
+			presence.getSetting<boolean>("buttons"),
+		]),
+		presenceData: PresenceData = {
+			largeImageKey:
+				"https://raw.githubusercontent.com/leonardssh/vscord/main/assets/icons/idle-vscode.png",
+			smallImageKey:
+				"https://raw.githubusercontent.com/leonardssh/vscord/main/assets/icons/vscode.png",
+			startTimestamp,
+		},
+		status: Stauts = {
+			file: document.querySelector(".tab.active a")?.textContent,
+			branch: document.querySelector("#status\\.scm\\.0")?.textContent.trim(),
+			error: document
+				.querySelector("#status\\.problems")
+				?.textContent.split("  ")[0],
+			problems: document
+				.querySelector("#status\\.problems")
+				?.textContent.split("  ")[1],
+			workspace: document
+				.querySelector("div.pane-header[aria-label='Folders Section'] h3")
+				?.getAttribute("title"),
+			editor: {
+				lang: document.querySelector("#status\\.editor\\.mode")?.textContent,
+				encoding: document.querySelector("#status\\.editor\\.encoding")
+					?.textContent,
+				selection: document.querySelector("#status\\.editor\\.selection")
+					?.textContent,
+			},
+		},
+		filename = status.file,
```

Comment:
Update referencds too
```suggestion
		{ file } = status,
```